### ISSUE TITLE
Add support for ETC1 and ETC2

### ISSUE THREAD

**Issue by Nominom** [Nominom/BCnEncoder.NET#7]

**Comment by onepiecefreak3** [comment]
I'm actually not sure about this one. While ETC is block based, it isn't part of the BCn series of image compressions. Would it really be ok to add it to a library called BCnEncoder?

**Comment by Nominom** [comment]
I don't know. This was more of a long-term plan for me if I wanted to add more formats. It is block-based like you said so it's still within the theme of Block Compression. I'm also not sure what the alternative libraries are for ETC, so I'm not sure how needed it would be. It could even be offered as an extension package, like BCnEncoder.Mobile or something. In any case BC6H support is way more of a priority, since it's the only one of the BCn lineup which this library does not support.

**Comment by onepiecefreak3** [comment]
Yes, I agree. Though I did much up until now, I think I can't really help with BC6H or ETC in the long run. At most I can maybe add decoding of BC6H? Depends on the specification I use, I guess.

**Comment by Nominom** [comment]
I think I would like to add BC6H encoding and decoding. I'm interested in learning it once I have more free time. You've been a big help already.
